packages/snaps-utils/src/manifest/validator-types.ts

```diff
@@ -0,0 +1,44 @@
+import type { SnapFiles, UnvalidatedSnapFiles } from '../types';
+import type { SnapManifest } from './validation';
+
+// TODO(ritave): Research using patch based fixing similar to eslint
+//               https://eslint.org/docs/latest/extend/custom-rules#applying-fixes
```

Comment:
Maybe open a ticket.

Reply:
This task is so monumental I don't think we'll move towards path based fixing unless we make a tool similiar to eslint. This TODO is left just as a note, I'll update it to a read-more